eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```
@@ -130,3 +131,28 @@ jobs:
           testRunTitle: "$(OSName) - Browser - Integration Tests - [Node $(NodeVersion)]"
         condition: and(succeededOrFailed(), eq(variables['TestType'], 'browser'))
         displayName: "Publish browser integration test results"
+
+  - job: "TestSamples"
+
+    variables:
+      - template: ../variables/globals.yml
+
+    pool:
+      vmImage: "$(OSVmImage)"
+
+    steps:
+      - template: ../steps/common.yml
+
+      - script: |
+          node common/scripts/install-run-rush.js install
+        displayName: "Install dependencies"
+
+      - script: |
+          node common/scripts/install-run-rush.js build -t "${{parameters.PackageName}}" --verbose
+        displayName: "Build libraries"
+
+      - script: |
+          node common/scripts/install-run-rushx.js execute:samples --verbose
```

I think this should be changed to 
```suggestion
          node ../../../common/scripts/install-run-rushx.js execute:samples --verbose
```

---

I think I'll try Brandon't suggestion first, in that way i won't have to change the directory etc

---

I changed the command to run the script `npm-run-command.js`  as suggested by Brandon, which reduces the dependency of an additional parameter